modules/building/pages/prefetching-dependencies.adoc

```diff
@@ -99,118 +99,62 @@ For more Go-specific `.netrc` details, see link:https://go.dev/doc/faq#git_https
-
-. Generate a fully resolved `requirements.txt` file that contains all the transitive dependencies and pins them to a specific version and hash by using the following command:
-
-+
-[source,command]
-----
-$ pip-compile pyproject.toml --generate-hashes
-----
 +
 [NOTE]
-==== 
-* To successfully run the previous command, your environment must be as close as possible to the environment in the container build. That is, the environment should have the same operating system and the same Python _$major.$minor_ version.
-
-* The previous command assumes that you have defined project dependencies in `pyproject.toml`. However, if you have defined the project dependencies in either the `setup.py`, `requirements.txt`, or `requirements.in` files, make sure you update the command accordingly.
 ====
-+
-. Add the `requirements.txt` file to the root of your component source code. 
-
-. In the root of your repository create a `requirements-build.in` file.
-
-. Copy the build system requirements from the `pyproject.toml` file to the `requirements-build.in` file.
-
-[start=10]
-. Run the `pip_find_builddeps.py` script and `pip-compile` the outputs by using the following command:
-
-+
-[source,command]
-----
-$ pip_find_builddeps.py requirements.txt \
---append \
---only-write-on-update \
--o requirements-build.in
-----
+* By default, Cachi2 processes `requirements.txt` and `requirements-build.txt` at a specified path.
+====
 
-. Use the `pip-compile` command to convert the `requirements-build.in` file in to the `requirements-build.txt` file by using the following command:
+.. Optional: In case the requirements files do not use the standard names, or in case more than a single requirements or requirements-build file is needed, you can use the following additional parameters:
 
 +
-[source,command]
+[source,yaml]
 ----
-$ pip-compile requirements-build.in --allow-unsafe --generate-hashes
+spec:
+    params:
+        -   ...
+        -   name: prefetch-input
+            value: '{"type": "pip", "path": ".", "requirements_files": ["requirements.txt", "requirements-extras.txt", "tests/requirements.txt", "requirements_build_files": ["other-build-requirements.txt"]}'
 ----
 
-. Add the `requirement-build.txt` file to your project. It does not require any changes to your build process. 
-
-+
-NOTE: `pip` automatically installs the build dependencies when needed for explicit installation. The purpose of the `requirement-build.txt` file is to enable Cachi2 to fetch the build dependencies and provide them to `pip` for offline installation in a network-isolated environment.
-
-+
-[NOTE]
-====
-* By default, Cachi2 processes `requirements.txt` and `requirements-build.txt` at a specified path.
+NOTE: To troubleshoot any issues you might experience when you enable prefetch builds for `pip` with source dependencies, see link:https://github.com/containerbuildsystem/cachi2/blob/main/docs/pip.md#troubleshooting[cachi2 documentation]
```

Comment:
you use hermeto links above. Do you want to be consistent here?

Reply:
Fixed.